masschallenge/impact-api
# AC-7476 Extend Office Hours Base Layout for Finalists

### Changes introduced in: [AC-7476](https://masschallenge.atlassian.net/browse/AC-7476):
- Added view logic
### How to test
- Schedule office hours for a finalist user
- Masquerade as that finalist
- Notice the new view that conforms to the ticket's done
- Notice that any other user see the message, "Waiting for content"
